demos/HST/S4_wfc3_template.ecf

```diff
@@ -27,29 +29,30 @@ maxiters        5       # The number of iterations of sigma clipping that should
 boundary        'fill'  # Use 'fill' to extend the boundary values by the median of all data points (recommended), 'wrap' to use a periodic boundary, or 'extend' to use the first/last data points
 fill_value      mask    # Either the string 'mask' to mask the outlier values (recommended), 'boxcar' to replace data with the mean from the box-car filter, or a constant float-type fill value.
 
-# Temporal binning
-sum_reads       True    # Sum together all of the reads from one scan?
-
 # Limb-darkening parameters needed to compute exotic-ld
-compute_ld      True
-inst_filter     Prism #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
-metallicity     0.1 #metallicity of the star 
-teff            6000 #effective temperature of the star in K
-logg            4.0 #surface gravity in log g
-exotic_ld_direc /home/User/exotic-ld_data/ #directory for ancillary files for exotic-ld
-exotic_ld_grid  3D #1D or 3D model grid
+compute_ld      False
+inst_filter     G102    # Filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
+metallicity     0.1     # Metallicity of the star 
+teff            6000    # Effective temperature of the star in K
+logg            4.0     # Surface gravity in log g
+exotic_ld_direc /home/User/exotic-ld_data/ # Directory for ancillary files for exotic-ld, download from: https://zenodo.org/record/6344946
+exotic_ld_grid  3D      # 1D or 3D model grid
+exotic_ld_file  #/home/User/exotic-ld_data/Custom_throughput_file.csv # Custom throughput file, for examples see Eureka/demos/JWST/Custom_throughput_files
+
+# Temporal Binning
+sum_reads       True    # Sum together all of the reads from one scan?
 
 # Diagnostics
-isplots_S4      5       # Generate few (1), some (3), or many (5) figures (Options: 1 - 5)
-vmin            0.97    # Sets the vmin of the color bar for Figure 4101.
-vmax            1.03    # Sets the vmax of the color bar for Figure 4101.
-time_axis       'y'     # Determines whether the time axis in Figure 4101 is along the y-axis ('y') or the x-axis ('x')
-hide_plots      False   # If True, plots will automatically be closed rather than popping up
+isplots_S4      3       # Generate few (1), some (3), or many (5) figures (Options: 1 - 5)
+vmin            0.97    # Sets the vmin of the color bar for Figure 3101.
+vmax            1.03    # Sets the vmax of the color bar for Figure 3101.
+time_axis       'y'     # Determines whether the time axis in Figure 3101 is along the y-axis ('y') or the x-axis ('x')
```

Comment:
I think you may have merged with an outdated main branch or something - these should be Figure 4101 still

Reply:
that was copy-pasted from the tests wfc3 .ecf file but changed both now